Gems/ROS2/Code/Source/FactorySimulation/ConveyorBeltComponent.cpp

```diff
@@ -61,7 +61,7 @@ namespace ROS2
 
     void ConveyorBeltComponent::Activate()
     {
-        AzPhysics::SystemInterface* physicsSystem = AZ::Interface<AzPhysics::SystemInterface>::Get();
+        [[maybe_unused]] AzPhysics::SystemInterface* physicsSystem = AZ::Interface<AzPhysics::SystemInterface>::Get();
```

Comment:
in each case, we don't actually need to run it in release builds. Please move to asserts, as in:
` AZ_Assert(AZ::Interface<AzPhysics::SystemInterface>::Get(), "No physics system");`
Apply to other cases.